src/Compilers/Core/Portable/Symbols/ISymbolInternal.cs

```diff
@@ -1,10 +1,14 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
+#nullable enable
+
 namespace Microsoft.CodeAnalysis.Symbols
 {
     internal interface ISymbolInternal : ISymbol
     {
+#nullable disable // Skipped for now https://github.com/dotnet/roslyn/issues/39166
```

Comment:
>disable [](start = 10, length = 7)

why disable here?

Reply:
It's not that I want to disable it here, but rather that I am not ready to enable it.